optimize/backend/src/main/java/io/camunda/optimize/tomcat/ResponseHeadersFilter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.tomcat;
+
+import io.camunda.optimize.service.util.configuration.ConfigurationService;
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class ResponseHeadersFilter implements Filter {
+
+  private final ConfigurationService configurationService;
+
+  public ResponseHeadersFilter(final ConfigurationService configurationService) {
+    if (configurationService == null) {
+      throw new IllegalArgumentException("configurationService cannot be null");
+    }
+
+    this.configurationService = configurationService;
+  }
+
+  @Override
+  public void doFilter(
+      final ServletRequest request, final ServletResponse response, final FilterChain chain)
+      throws IOException, ServletException {
+    final HttpServletResponse httpResponse = (HttpServletResponse) response;
+
+    final var headers =
```

Comment:
I think in the mono repo we can use `var` if needed. Not to say this can't be changed, just that there is less strictness around this than in the Optimize repo

Reply:
I don't like using "var" very much, I was trying to be quick here but then I forgot to change it.
Thanks for noticing